app/models/katello/content.rb

```diff
@@ -5,15 +5,12 @@ class Content < Katello::Model
     has_many :products, :through => :product_contents
     belongs_to :organization, :inverse_of => :contents, :class_name => "::Organization"
 
-    validates :label, :uniqueness => {:scope => :organization_id}
-    validates :cp_content_id, :uniqueness => {:scope => :organization_id}
```

Comment:
Removed the validations to rely on DB constraints: TODO: Verify we have db constraints that satisfy these

Reply:
No db constraints present, added in migration

Reply:
I can't remember what made me go this route because at the time I did test with constraints in the db ...